plugin/trino-hive/src/main/java/io/trino/plugin/hive/RcFileFileWriter.java

```diff
@@ -158,6 +158,18 @@ public void commit()
                 throw new TrinoException(HIVE_WRITE_VALIDATION_FAILED, e);
             }
         }
+
+        Callable<Void> rollbackAction = this.rollbackAction;
+        return new RollbackAction(
```

Comment:
This boilerplate in all of the writers is unnecessary. We should either remove `RollbackAction` and use `Callable<Void>` or change the writers to take `RollbackAction` directly, so we don't have to adapt.